### ISSUE TITLE
Australian 3G shutdown forces newer model graphene phones offline

### ISSUE THREAD

**Issue by Jonnybeez** [GrapheneOS/os-issue-tracker#4280]
Hi All, 
  Australia turned off their 3G network on the 4th of November, since then all graphene phones running on m2m or iot cards cannot resolve a connection even though they have 4 and 5G capability.

Older versions of graphene work such as pixel 3 and 4 and are able to jump to 4G no problem but i have tested 6,7,8,9 all not working.

the sims used in the phones work in 4G dongles and other devices also it just seems to be newer model updated graphene phones and also android phones running 13+

If anyone can shed some light I would love some help.

Thank you.

- Jonathan

**Comment by Jonnybeez** [comment]
> It works fine on the stock OS?

It works fine on the GSM, main network but m2m and iot cards it doesnt.

however pixel 3s and 4s running older images do work on the m2m and iot networks its strange

**Comment by Jonnybeez** [comment]
> Is there any functionality that's broken with GrapheneOS but not broken with the stock OS?

nothing obvious everything seems ok, its difficult to determine if its even OS based or if there is some kind of issue on the sim side that is blocking newer phones. Its curious that the older models that havent updated work fine and the newer ones dont. even in network settings if 4G only is selected i feel like its still looking for 3G (which isnt there anymore) and looping out. It may also be because 3G is reserved for emergency lines now and the network issues some kind of denial to the phone im not sure im trying to figure it out.

**Comment by Jonnybeez** [comment]
> The issue isn't the 3G shutdown but rather Australia banning using imported phones on the networks. It's an intentional block. It doesn't appear there's much we can do about this. If it's broken on the stock OS, there isn't an issue with GrapheneOS.

Yeah i read that also but in practice it doesnt seem to be the case I have plenty of OS phones still working and plenty of australian imei phones not working. I have purchased some yesterday from a large local retailer australian models and they also dont work its very confusing.

**Comment by Jonnybeez** [comment]
Yeah it is ridiculous and haphazard. I assumed 3G devices would be impacted but thought the newer devices would be fine but its really broken a lot more than i was expecting. Thanks for the advice anyway.

**Comment by Jonnybeez** [comment]
> The issue is likely that you have international Pixel variants and their IMEI is not whitelisted. GrapheneOS would likely work perfectly fine for you on a whitelisted Pixel. You need an Australian Pixel that's unlocked. Considering how bad the carriers are in this regard, they likely lock their devices, so you'd need a Google Store Australia Pixel.

I have a pixel 9 pro that was ordered from googles au online store (australian stock) it works on the public network fine but on the iot network it doesnt, however i also have pixel 4a's and 3's and they all work fine on both networks thats the only thing that leads me to believe its something to do with the later updates of both android and graph. it 'feels' like the newer devices look for 3g and dont even try 4 or 5g when they cant find it. So either the public network doesnt have a whitelist and the iot network does or the android update differs from older versions with regard to the netcode.

Just very frustrating as you can imagine and not enough information is offered from the carriers. Anyway looks like ill have to adapt. Thank you my friend i appreciate your time. If i find anything more concrete ill update the thread for others.

**Comment by FetchFast** [comment]
I'm trying to find a YouTube video that detailed this. It seems to be a combination of incompetence, anti competitive monopoly practice mixed up with efforts to address foreign spying.

From memory, they've switched off both 2g and 3g and this causes problems with how _some_ phones handle connecting, including emergency calls. This includes brand new phones. Is grapheneos any different.

They've also started using a white list TAC on imeis. This screws up more phones.

Also, some phones will connect and start to work, but then get blocked on (another?) imei blacklist. I can't remember what the exact conditions for this were.

It's all very confusing. The blacklisting makes it harder to diagnose.

It seems imei is a weak point that legally grapheneos can't help with. But is there an aspect to this that grapheneos can help with? It's helpful to see this situation exposed as it has been for others elsewhere to be warned
